subsys/bluetooth/controller/ll_sw/nordic/lll/lll_conn.c

```diff
@@ -246,6 +253,17 @@ void lll_conn_isr_rx(void *param)
 	is_empty_pdu_tx_retry = lll->empty;
 	lll_conn_pdu_tx_prep(lll, &pdu_data_tx);
 
+#if defined(CONFIG_BT_CTRL_DF_CONN_CTE_TX)
+	if (pdu_data_tx->cp == PDU_DATA_CTE_PRESENT_BIT_ENABLED) {
+		lll_df_conn_cte_tx_enable(&lll->df_tx_cfg);
+
+		cte_len = CTE_LEN_US(pdu_data_tx->cte_info.time);
+	} else
+#endif /* CONFIG_BT_CTRL_DF_CONN_CTE_TX */
+	{
+		cte_len = 0;
```

Comment:
Should we do this for all unsigned values? 

```suggestion
		cte_len = 0U;
```

Reply:
corrected